R/xl_open.R

```diff
@@ -115,7 +113,7 @@ chooseExcelApp <- function() {
     unnprog <- unname(availProg)
     message(sprintf("Setting options(openxlsx2.excelApp = '%s')", unnprog))
     options(openxlsx2.excelApp = unnprog)
-    invisible(unnprog)
+    return(invisible(unnprog))
```

Comment:
needed the return, otherwise it would always end on the stop at the end of the function